### PR TITLE
add JCasC test

### PR DESCRIPTION
**Pull request by casz**

### CODE REVIEW

File: src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import static org.junit.Assert.assertNotNull;
+
+public class ConfigAsCodeTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void should_support_jcasc_from_yaml() throws Exception {
+        ConfigurationAsCode
```
Comment (timja) [inline src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java]: use ConfiguredWithCode annotation / rule?
e.g.
https://github.com/jenkinsci/azure-keyvault-plugin/pull/15/files

Comment (casz) [reply src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java]: I cannot it is not in packaged

Comment (timja) [reply src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java]: seems to work in that PR, you need to add another dep with the tests classifier

Comment (casz) [reply src/test/java/jenkins/metrics/api/ConfigAsCodeTest.java]: Or at least I thought so

**Comment by casz** [comment]
Superseded by #46

**Pull request closed without merging**